project/scripts/sbt

```diff
@@ -1,26 +1,20 @@
 #!/usr/bin/env bash
 
-CI_TEST=$1
+# Usage:
+# ./sbt <cmd>
 
-if [ -n "$2" ]; then
-  if [ $2 = true ]; then
-    SHOULD_RUN=true
-  else
-    SHOULD_RUN=false
-  fi
-else
-  SHOULD_RUN=true
+CMD="$1"
+
+if [ -z "$CMD" ]; then
```

Comment:
Nitpick, this should be:

```bash
if [ -z ${CMD+x} ]; then
```

otherwise you can pass empty string here and it will accept it as valid input.

Reply:
My understanding is that it is the other way around:
```shell
> FOO=""; if test -z ${FOO+x}; then echo "Empty"; else echo "Full"; fi
Full
> FOO=""; if test -z "$FOO"; then echo "Empty"; else echo "Full"; fi
Empty
```

Reply:
Maybe I wasn't clear - you're testing if the variable `FOO` is empty string - instead of testing if the variable is set at all. E.g. setting `FOO=""` versus not setting it at all

:)

Reply:
But seriously though, nitpick 😛

Reply:
I want the script to exit if the variable is not set or if it is an empty string. An empty cmd should be reject IMO. With `-z "$CMD"`, it rejects both cases.